Source/Particles/SpeciesPhysicalProperties.H

```diff
@@ -33,12 +34,20 @@ namespace species
             return PhysicalSpecies::positron;
         if( species=="photon" )
             return PhysicalSpecies::photon;
-        if( species=="hydrogen" )
-            return PhysicalSpecies::hydrogen;
-        if( species=="proton" )
+        if( species=="neutron" )
+            return PhysicalSpecies::neutron;
+        if( (species=="hydrogen") || (species=="proton") )
```

Comment:
Note that this is not very consistent with the way in which we treat "helium". 
For consistency, we should distinguish:
- hydrogen (includes the mass of the electron, averages over isotopes abundance)
- proton/hydrogen1 (does not include the mass of the electron, does not average over isotopes)
However, in order to preserve the code behavior, I did not introduce this change.

Reply:
Agreed. Didn't we discuss this for #3090 that @lucafedeli88 has open? :)

Reply:
Ah, yes. Actually the present PR duplicates #3090. I will close it for now.